src/tmx/TmxUtils/src/DsrcBuilder.cpp

```diff
@@ -35,11 +35,8 @@ void DsrcBuilder::AddTimAdvisory(TiDataFrame *frame, unsigned int speedLimit)
 void DsrcBuilder::AddItisCode(ITIScodesAndText *advisory, long code)
 {
 	ITIScodesAndText__Member* member = (ITIScodesAndText__Member*)malloc(sizeof(ITIScodesAndText__Member));
-	#if SAEJ2735_SPEC < 2020
-	member->item.present = ITIScodesAndText__Memberitem_PR_itis;
-	#else
 	member->item.present = ITIScodesAndText__Member__item_PR_itis;
```

Comment:
The original statement indicates that J2735 spec before 2020 uses  member: ITIScodesAndText__Memberitem_PR_itis; Can I assume that 2016 is also using this member

Reply:
This was a bug in the previous implementation of 2016. The current code works as is for 2016